templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,19 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 5.0
 
+**Feature level 118**
+
+* [`GET /messages`](/api/get-messages), [`GET /events`](/api/get-events):
+  Updated `edit_history` data structure of message
+  objects. Entries for stream edits now include a both a `prev_stream` and
+  `stream` field to indicate the previous and current stream IDs. Entries
+  for topic edits now include both a `prev_topic` and `topic` field to
+  indicate the previous and current topic. The legacy `prev_subject` field
+  was renamed to `prev_topic` and is now deprecated.
+
+* [`GET messages/{message_id}/history`](/api/get-message-history):
+  Added `stream` field to message history `snapshot` objects to indicate
+  the updated stream ID of the message if it was modified.
```

Comment:
I updated the zulip.yaml stuff to use feature level 118 (was still on 116), and also did some editing to how we describe fields to use phrases like "immediately prior to this edit" which I hope will be clearer.

Reply:
🤔 Hmmm... I thought I had updated the changelog for that feature level. Thanks for updating that!

I agree that the "immediately prior to this edit" phrasing makes the edit log timing clearer.